ansible/tasks/oauth2-proxy.yml

```
@@ -43,3 +43,28 @@
     name: "oauth2-proxy"
     enabled: true
     state:  "started"
+
+# Same thing but now for admin only oauth access
+
+- name: "copy allowed users file"
+  template:
+    src: "templates/etc/oauth2_proxy/allowed-users"
+    dest: "/etc/oauth2_proxy/allowed-users"
```

We probably also want to restart the proxy if this list of allowed users changes

```suggestion
    dest: "/etc/oauth2_proxy/allowed-users"
  notify: "restart oauth2-proxy-admin"
```